### ISSUE TITLE
Cannot retrieve a mutable collider from the world

### ISSUE THREAD

**Issue by seppo0010** [rustsim/nphysics#167]
https://www.nphysics.org/rustdoc/nphysics3d/world/struct.World.html?search=#method.collider

According to the documentation
```
pub fn collider(&self, handle: ColliderHandle) -> Option<&Collider<N>>

Get a mutable reference to the specified collider.

Returns None if the handle does not correspond to a collider in this world.
```

The reference is definitely not mutable. There should probably need to be a different method for this purpose, probably with this signature:

```
pub fn collider_mut(&mut self, handle: ColliderHandle) -> Option<&mut Collider<N>>
```

**Comment by sebcrozet** [comment]
@seppo0010 Thank you for seeing this.
This will be fixed in the next release of v0.10 tomorrow.

**Comment by seppo0010** [comment]
Fixed. Thanks!
